javascript/javascript2/week3/Warmup/Warmup.js

```diff
@@ -0,0 +1,84 @@
+console.log("Script Loaded!!");
+
+function consoleLogTime(delay = 0, stringToLog = "") {
```

Comment:
You don't need to define the arguments here, `(delay, stringToLog)` is good enough.